daemon/algod/api/server/v2/handlers.go

```diff
@@ -238,6 +238,14 @@ func (v2 *Handlers) AccountInformation(ctx echo.Context, address string, params
 	if err != nil {
 		return internalError(ctx, err, errFailedLookingUpLedger, v2.Log)
 	}
+	block, _, err := myLedger.BlockCert(basics.Round(lastRound))
```

Comment:
My first time using the ledger. I'm not sure this is the ideal way of doing so to retrieve `minBalance`

Reply:
it doesn't look that you really need the block. You only need the block header. try using `myLedger.BlockHdr`

Reply:
👍